app/views/deployments.html

```diff
@@ -26,6 +26,45 @@
       <div class="middle-content">
         <div class="container-fluid">
           <alerts alerts="alerts"></alerts>
+          <!-- TODO: keep statefulSets here or put on own page? -->
+          <div class="row">
+            <div class="col-md-12">
+              <h3>StatefulSets</h3>
+              <table class="table table-bordered table-hover table-mobile">
+                <thead>
+                  <tr>
+                    <th>Name</th>
+                    <th>Generation</th>
```

Comment:
a lot of objects have Generation and we don't usually show it, I would probably just go with Name, Status, Created for now, we can discuss other options during design review thurs.